### ISSUE TITLE
Support multi-lingual

### ISSUE THREAD

**Issue by LittleFish-233** [DAXGRID/typesense-dotnet#138]
For now, multilingual is supported in version 0.24. Whether the language can be specified when the collection is created？
Like this:
```csharp
 var schema = new Schema(
   "Addresses",
     new List<Field>
     {
         new Field("id", FieldType.String, false),
                new Field("houseNumber", FieldType.String, false,"zh"),
            },
            "houseNumber");

        var createCollectionResponse = await typesenseClient.CreateCollection(schema);
        Console.WriteLine($"Created collection: {JsonSerializer.Serialize(createCollectionResponse)}");
```

Version 0.24.0: https://github.com/typesense/typesense/releases/tag/v0.24.0

> Multi-lingual support: Chinese, Japanese, Korean, Cyrillic and Thai are now supported via the use of field-level
`locale` flag in the collection schema.

**Comment by runeanielsen** [comment]
Hi @LittleFish-233,

It is currently not supported in the client. I mainly work on this client library in my spare time, so I don't always have the time to keep up with all the features, so I try to implement them when they're requested.

I'll add the support for it in a few days and let you know when it is done. :)

// Rune

**Comment by LittleFish-233** [comment]
Thank you so much @runeanielsen !

**Comment by runeanielsen** [comment]
Hi @LittleFish-233,

I've now added multilingual support. This has now been released in version `6.3.0`.

Here is an example of setting a field named "name" to Chinese.

```c#
var schema = new Schema(
    "my_collection_name",
    new List<Field>
    {
        new Field(
            "name",
            FieldType.String,
            locale: "zh"),
    });
```

// Rune
